### DISCUSSION TITLE
Custom FBW aileron not functioning

### DISCUSSION THREAD

**Discussion by Manochvarma** [flybywiresim/aircraft#2948] [Help] [answered]
I am using custom FBW latest version. I am unable to turn the aircraft (aileron) using the Numpad 4 and 6 key. I checked the key map and everything was as the default one. Elevator and rudder working well.  The sidestick not moving left and right when I press num 4 and 6. Why is it so?

**Comment by Inaxair** [comment, marked as answer]
Hello,

Do you have any other mods installed? If yes, try to relocate them temporaly for looking if it's a compatibility issue.
Do you have anything else plugged in the computer other then a mouse and a keyboard?
Which version of the sim/aircraft do you have installed?

Thanks
Inaxair from FlyByWire Simulation

**Reply by Manochvarma** [reply]
> Hello,
> 
> Do you have any other mods installed? If yes, try to relocate them temporaly for looking if it's a compatibility issue.

I don't have any other addon or mods installed. The only addon installed is A32NX. I have no problem with the Master version. The problem only exist in Custom FBW.
 
> Do you have anything else plugged in the computer other then a mouse and a keyboard?

Only thing plugged in is my notepad cooler.
 
> Which version of the sim/aircraft do you have installed?

1.12.13. Custom FBW A32NX and no any other addon installed.

**Reply by kdullabh** [reply]
Hi Dev team, I'm also having this same issue on the lastest Dev build 8f32381 of the FBW, no other mods installed just the A32NX.
I only use the keyboard and mouse and my headset plugged in.
Installed latest version of MSFS.
Appreciate if you can assist  with the work around.

**Reply by aguther** [reply]
https://docs.flybywiresim.com/start/autopilot-fbw/

Known issue due to an issue in the SDK.
